hydra-plutus/src/Hydra/Contract/Initial.hs

```diff
@@ -37,13 +38,12 @@ import PlutusTx (CompiledCode)
 import qualified PlutusTx
 import qualified PlutusTx.AssocMap as AssocMap
 import qualified PlutusTx.Builtins as Builtins
-import Hydra.Contract.Error (ToErrorCode (..))
 
 data InitialRedeemer
   = ViaAbort
   | ViaCommit
-      { -- | Points to the committed Utxo.
-        committedRef :: Maybe TxOutRef
+      { committedRef :: Maybe TxOutRef
+      -- ^ Points to the committed Utxo.
```

Comment:
My version of fourmolu was doing it. I hope we can upgrade the included version of our development setup soon as well.

Shall I revert them?

Reply:
No, unfortunately not easily. It depends on newer Cabal, which depends on newer base, which depends on newer GHC versions.

Reply:
I would revert to the original doc formatting so that we avoid future commit that will just reformat the old way because people are using the nix version of it.